spec.bs

```diff
@@ -6520,11 +6541,52 @@ Each {{InterestGroupScriptRunnerGlobalScope}} has a
 <div algorithm>
 The <dfn attribute for=InterestGroupScriptRunnerGlobalScope>privateAggregation</dfn> [=getter
   steps=] are:
-
   1. Return [=this=]'s [=relevant global object=]'s
     [=InterestGroupScriptRunnerGlobalScope/private aggregation=].
 </div>
 
+<div algorithm>
+The <dfn attribute for=InterestGroupScriptRunnerGlobalScope>protectedAudience</dfn> [=getter
+  steps=] are:
+  1. Return [=this=]'s [=relevant global object=]'s
+    [=InterestGroupScriptRunnerGlobalScope/protected audience utilities=].
+</div>
+
+<div algorithm>
+The <dfn method for="ProtectedAudienceUtilities">encodeUtf8(USVString input)</dfn> method steps are:
+1. Follow the steps of {{TextEncoder/encode(input)|TextEncoder.encode(input)}}.
+
+</div>
+
+<div algorithm>
+The <dfn method for="ProtectedAudienceUtilities">decodeUtf8(Uint8Array bytes)</dfn> method steps are:
+1. Let |decoder| be a new instance of [=UTF-8=]'s [=decoder=].

```

Comment:
do you think we can link to https://encoding.spec.whatwg.org/#utf-8-decoder?

Reply:
Done